### ISSUE TITLE
Hold Mission Duration conflict?

### ISSUE THREAD

**Issue by OzanKurt** [jkroepke/2Moons#282]
https://github.com/jkroepke/2Moons/blob/master/includes/pages/game/ShowFleetStep3Page.class.php#L399

I think the line should be so:

```php
$fleetEndTime		= $fleetStartTime + $duration;
```

Any idea?

**Comment by jkroepke** [comment]
No, because EndTime is `starttime + duration + holdtime + duration`

**Comment by onomato93** [comment]
jkroepke, i think that its wrong, because when you send fleet on hold mission, (from 1 planet to 2 planet u have 1 hour), and fleet came on target planet, wait 5-10min, after that turn around your fleet, fleet should back in 1hour, but your fleet need 1h + 5-10min (time of hold). Its wrong, because your fleet should have the same time of arriving.

**Comment by OzanKurt** [comment]
My reason for creating this issue was that the fleets having wrong arrival times.

For example: When I send a mission with 1 hour delay to a 15 minute planet.

it goes in 1:15 and comes back in 1:15.

In total 2:30. Should have been 1:30.

**Comment by onomato93** [comment]
but what i said is bad calculation too, because when you send fleet on holding mission, and flying duration is 1hour, and holding should been 20min, so when you turn back your fleet after 10min of holding, your time of back to planet should be again 1hour, but now is 1h + 10min. Have you any idea how to change it? I have no idea how to fix it, should be something like 
if you turn back your fleet from holding (1h time of flying + 10min/20min of holding)  your time should be again 1h. But if you change fleetEndTime, you broke holding time ;x

**Comment by jkroepke** [comment]
I will check this.
